docker/Dockerfile

```diff
@@ -18,6 +18,11 @@ ARG PYTHON_VERSION=3.8
 ###
 FROM docker.io/python:${PYTHON_VERSION}-slim as builder
 
+LABEL org.opencontainers.image.url='https://matrix.org/docs/projects/server/synapse'
+LABEL org.opencontainers.image.documentation='https://github.com/matrix-org/synapse/blob/master/INSTALL.md'
```

Comment:
I'm not 100% sure if these are the URLs we want to use? Do other people have opinions?

Reply:
I wasn't sure neither. Finding something that looks official using Google isn't exactly easy.

Reply:
We may want to link to the docker README instead of the general INSTALL.md guide for the docker containers?

https://matrix.org/docs/projects/server/synapse is probably fine, especially since we have a link to the github in another line. We could probably do to make that page a bit prettier though.

Reply:
```suggestion
LABEL org.opencontainers.image.url='https://matrix.org/docs/projects/server/synapse'
LABEL org.opencontainers.image.documentation='https://github.com/matrix-org/synapse/tree/master/docker'
```